### PR TITLE
Some changes, no additions

### PR DESCRIPTION
**Pull request by alexture**
- Slight cleanups of promotional text
- Reorganisation of the "Other" section as most of that stuff belonged somewhere more specific
- I lied - I added the French branch :)

### CODE REVIEW

**Comment by mbiesiad** [comment]
Hi @alexture Wow, awesome! 🚀 I will make a review ASAP in the coming days ✨

File: README.md [hunk 1]
```diff
@@ -22,13 +22,14 @@ This list contains tools, books, articles, blogs, courses and everything related
 - [Polish](https://github.com/mbiesiad/awesome-translations/tree/pl_PL)
 - [Spanish](https://web.archive.org/web/20220511155347/https://github.com/JoseDeFreitas/awesome-translations/tree/es_ES)
 - [Italian](https://github.com/niedev/awesome-translations/tree/it_IT)
+- [French](https://github.com/alexture/awesome-translations/blob/fr_FR/README.md)
```
Comment (mbiesiad) [inline README.md:25]: Awesome work! Looks great ❤ But here will be better to unified format like without `/blob`, ie. `https://github.com/alexture/awesome-translations/tree/fr_FR` I think

File: README.md [hunk 2]
```diff
@@ -22,13 +22,14 @@ This list contains tools, books, articles, blogs, courses and everything related
...
 
 ## Platforms
 
 ### Localization and translation platforms
 
 - [Crowdin](https://crowdin.com/) - Closed source cloud-based localization technology and services company.
-- [LocaleApp](https://www.localeapp.com/) - Effective app localization and translation.
+- [LocaleApp](https://www.localeapp.com/) - App localization and translation.
```
Comment (mbiesiad) [inline README.md:32]: Good point! There is missing part I think; suggested: `App for`

File: README.md [hunk 3]
```diff
@@ -39,7 +40,7 @@ This list contains tools, books, articles, blogs, courses and everything related
 ### Project management
 
 - [Zanata](http://zanata.org/) - Web-based translation platform for translators, content creators and developers to manage localisation projects.
-- [Traduora](https://github.com/traduora/traduora) - A platform for manage translation workflow in teams.
+- [Traduora](https://github.com/traduora/traduora) - A platform to manage translation workflow in teams.
```
Comment (mbiesiad) [inline README.md:43]: Good point! 🎉 Thanks a lot! ✨

**Review by mbiesiad — COMMENTED** [review]
Good points! ✨

**Comment by alexture** [comment]
Both issues should be fixed now!

**Review by mbiesiad — APPROVED** [review]
LGTM 👍

**Comment by mbiesiad** [comment]
Hi @alexture you're just awesome! 🚀 Thank you so much again for your effort and contributions here ✨ Sorry for delay, thank you for your patience ❤ Nice changes, great work as always! Approved, merged 🎉 Best wishes,

**Pull request merged**